src/model/create-model.ts

```diff
@@ -0,0 +1,62 @@
+import { find } from '../handler/find';
+import { remove } from '../handler/remove';
```

Comment:
It seems that `remove` is imported from `handler/remove` but never used, are we planning on using this for future code written in this file?

Reply:
Fixed: all unused variables were removed